skimage/exposure/tests/test_histogram_matching.py

```diff
@@ -27,8 +27,6 @@ class TestMatchHistogram:
     image_rgb = data.chelsea()
     template_rgb = data.astronaut()
 
-    # To handle with mutlichannel ==False
-    #(image_rgb[:, :, 0], template_rgb[:, :, 0]),
     @pytest.mark.parametrize('image, reference', [
         (image_rgb, template_rgb)
     ])
```

Comment:
I feel like this issue could be more easily solved with:

```python
@pytest.mark.parametrize('image, reference, multichannel', [
    (image_rgb, template_rgb, True),
    (image_rgb[..., 0], template_rgb[..., 0], False),
])
def test_match_histograms(image, reference, multichannel):
    ...
    matched = exposure.match_histograms(image, reference, multichannel=multichannel
```

Would that fix the problem or am I missing some other differences?

Reply:
Oh! Thank you for pointing it out. :smile: I was not familiar with parameterized testing. It's good to learn something new.